funds-manager/funds-manager-server/src/metrics/cost.rs

```diff
@@ -228,6 +235,21 @@ impl MetricsRecorder {
         Ok(price)
     }
 
+    /// Get the cost of the given amount of WEI in USD
+    async fn get_wei_cost_usdc(&self, wei: U256) -> Result<f64, FundsManagerError> {
+        // We use the weth price here as a stand in for the eth price seeing as weth
+        // does not trade at a discount to eth. As well, we fetch the price using the
+        // arbitrum one chain for simplicity -- the price will be the same across chains
+        let weth = Token::from_ticker("WETH");
```

Comment:
let's be sure to use `Token::from_ticker_on_chain("WETH", Chain::ArbitrumOne)` here - this ensures that the `Token` constructed by the `PriceReporterClient` using `weth.get_alloy_address()` matches the one we constructed here.

not strictly necessary currently - it wouldn't result in any errors if we construct the Base WETH token and try getting its price while passing in `Chain::ArbitrumOne` - but helps protect against regressions if we use the `chain` parameter for anything else in `get_price`